enterprise/j2ee.sun.ddui/src/org/netbeans/modules/j2ee/sun/share/plan/DeploymentPlan.java

```diff
@@ -255,16 +255,16 @@ public void dump(StringBuffer str, String indent){
 		Object o;
 		org.netbeans.modules.schema2beans.BaseBean n;
 		str.append(indent);
-		str.append("FileEntry["+this.sizeFileEntry()+"]");	// NOI18N
+		str.append("FileEntry[").append(this.sizeFileEntry()).append("]");	// NOI18N
```

Comment:
changes like this are really unnecessary. Since it would compile to append anyway on 8, and the faster invokedynamic+StringConcatFactory version on 9+. It doesn't improve readability and might reduce performance -> lets not change it ;)

explained here https://github.com/apache/netbeans/pull/3241#issuecomment-962668861